rust/processor/src/db/mod.rs

```diff
@@ -1 +1,2 @@
-pub mod common;
```

Comment:
re folder structure and naming: should we keep `common` module to contain all the `raw models` and the `postgres`/`parquet` module to contain the the `convertible` trait implementations? it would make imports cleaner so the parquet processors don't import from `db::postgres::models::`

Reply:
good callout! it's better to have raw table items in the `common` module.